### PR TITLE
Add appveyor

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

**Comment by henrykironde** [comment]
These are tests for packaging mainly. 
We try to make sure that `R CMD build` and `R CMD check` are working on windows.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        Import-Module '..\appveyor-tool.ps1'
+
+install:
+  ps: Bootstrap
+
+cache:
+  - C:\RLibrary
+
+environment:
+  NOT_CRAN: flase
```
Comment (ethanwhite) [inline appveyor.yml:17]: Looks like a typo here. Should be "false"?

Comment (henrykironde) [reply appveyor.yml:17]: I was trying to test `NOT_CRAN: false` and `NOT_CRAN: True`. I did not see any change in the way the checks are tested. I may be missing something on how the that tag is used.

Comment (henrykironde) [reply appveyor.yml]: I have set it back to `NOT_CRAN: true`

**Comment by ethanwhite** [comment]
:+1: LGTM. Is this ready to merge?

**Comment by henrykironde** [comment]
I could me missing something. I expected appveyor to be part of the tests. I did add the project to appveyor so we should be seeing the tests. I guess you may need to log into appveyor and try to add weecology/rdataretriever

**Comment by henrykironde** [comment]
@ethanwhite , I did log into appveyor and added the repo under  ropensci(just deleted it).
The addition should have let to firing up the tests when a PR is made.
Could you try adding rdataretriever under the  ropensci repos in appveyour, we shall have to change the badge url

**Comment by ethanwhite** [comment]
I think this is turned on now. Can you try pushing a small change to the PR to see if it triggers successfully?

**Comment by henrykironde** [comment]
Thanks @ethanwhite, you will have to change the url badge because I cannot access it. 
`[AppVeyor build status](https://ci.appveyor.com/project/henrykironde/rdataretriever)`

**Comment by henrykironde** [comment]
You could also paste it here so that I make the change

**Comment by ethanwhite** [comment]
`[Build status](https://ci.appveyor.com/project/ethanwhite/rdataretriever)`

**Comment by ethanwhite** [comment]
Just made the change. Merging.

**Pull request merged**